core/datastore/src/commonMain/kotlin/org/mifos/mobile/core/datastore/model/AppSettings.kt

```
@@ -0,0 +1,9 @@
+package org.mifos.mobile.core.datastore.model
+
+data class AppSettings(
+    val tenant: String?,
+    val baseUrl: String?,
+    val passcode: String?,
+    val appTheme: Int?,
```

use enum class directly, all property except passcode shouldn't be null, create companion object to provide default value

---

and it does not contain appLanguage property